code/cobre.py

```diff
@@ -15,8 +15,30 @@
 import argparse
 from pathlib import Path
 
-
-def process_data(csv_file_p, output_json_p):
+# Define metadata
+metadata = {
+    "participant_id": {
+        "original_field_name": "[none given]",
+        "description": "Unique identifier for each participant",
+    },
+    "age": {
+        "original_field_name": "Current Age",
+        "description": "Age of the participant in years",
+    },
+    "sex": {"original_field_name": "Gender", "description": "Sex of the participant"},
+    "site": {
+        "original_field_name": "[none given]",
+        "description": "Site of imaging data collection",
+    },
+    "diagnosis": {
+        "original_field_name": "Subject Type",
+        "description": "Diagnosis of the participant",
+        "labels": {"CON": "Control", "SZ": "Patient"},
```

Comment:
sorry for going back and forth in this one - I just realised in bids this is called levels rather than labels. we might want to standardise that